docs/settingUp.md

```diff
@@ -32,12 +40,16 @@ More information can be found at [this documentation](https://help.github.com/ar
    ```sh
    ./gradlew appengineDownloadSdk
    ```
+   
+ **Verification** : Check your Gradle folder (the directory can be found with the command ./gradlew printUserHomeDir). A folder        appengine-sdk should be present.
```

Comment:
- with the command `./gradlew printUserHomeDir`
- A folder **named** `appengine-sdk` should be present.